certbot-apache/certbot_apache/parser.py

```diff
@@ -41,11 +43,21 @@ def __init__(self, aug, root, vhostroot=None, version=(2, 4),
         # issues with aug.load() after adding new files / defines to parse tree
         self.configurator = configurator
 
+        # Initialize augeas
+        self.aug = None
+        self.save_notes = ""
```

Comment:
Since `save_notes` are used with the reverter for the purposes of describing a checkpoint, what do you think about moving this string to the configurator object?

Reply:
Yeah, you are right. The `save_notes` is now moved to ApacheConfigurator.